modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/test/java/dev/galasa/framework/api/ras/internal/routes/TestRunQuery.java

```diff
@@ -81,11 +81,25 @@ private Map<String, String[]> setQueryParameter (Integer page, Integer size,Stri
 		return parameterMap;
 	}
 
+	private Map<String, String[]> setQueryParameter (Integer page, Integer size,String sort, String runname, String requestor, Integer agemin, Integer agemax, String group){
+		Map<String, String[]> parameterMap = new HashMap<String,String[]>();
+		parameterMap = addQueryIntParameter(parameterMap, "page", page);
+		parameterMap = addQueryIntParameter(parameterMap, "size", size);
+		parameterMap = addQueryParameter(parameterMap, "sort", sort);
+		parameterMap = addQueryParameter(parameterMap, "runname", runname);
+		parameterMap = addQueryParameter(parameterMap, "requestor", requestor);
+		parameterMap = addQueryTimeParameter(parameterMap, "from", agemin);
+		parameterMap = addQueryTimeParameter(parameterMap, "to", agemax);
+		parameterMap = addQueryParameter(parameterMap, "group", group);
+		return parameterMap;
+	}
```

Comment:
If this function is overloading the existing one with an added `group` parameter, we could reduce the duplication a bit by calling this function from the existing one, like:
```java
private Map<String, String[]> setQueryParameter (Integer page, Integer size,String sort, String runname, String requestor, Integer agemin, Integer agemax, String group){
    return setQueryParameter(page, size, sort, runname, requestor, ...);
}
```

Or instead of making a new function, we could just add the group query parameter to the tests that use it with `addQueryParameter(parameterMap, "group", group)`, which might be easier